src/pages/registration/references_tab/sub_type_forms/JournalForm.tsx

```diff
@@ -110,11 +110,14 @@ const JournalForm: FC = () => {
         </Field>
       </StyledArticleDetail>
 
-      <NviValidation
-        isPeerReviewed={publicationInstance.peerReviewed}
-        isRated={!!publicationContext?.level}
-        dataTestId="nvi_journal"
-      />
+      {(publicationInstance.type === JournalType.ARTICLE ||
+        publicationInstance.type === JournalType.SHORT_COMMUNICATION) && (
```

Comment:
venter avklaring fra Hege på dette, siden det er umulig for ShortCommunication å inngå i NVI nå, da det ikke er noe valg for peer reviewed der (som kreves for å inngå i NVI).

Reply:
svar: 
> ja, en må også kunne velge PeerReviewed for Short communication. (Denne er egentlig en undertype av Journal article, og er spilt inn av referansegruppen for NVA. Det er mulig at Sort communication på et senere tidspunkt kan slås sammen med Journal article, men det krever en opprydding i retningslinjer/rutiner for NVI-rapportering)